projects/documentation/scripts/build-search-index.js

```diff
@@ -164,7 +164,7 @@ async function main() {
         // Configure the index fields
         this.ref('metadata');
         // Boost title field for higher relevance when matching titles
-        this.field('title', { boost: 10 });
+        this.field('title', { boost: 100 });
```

Comment:
Adding more to the dialog docs made dialog, dialog base, and dialog wrapper have higher result than base docs, so we boosted the value of a matching title.